ansible/roles/products/common/resources/templates/scripts/setup-hand.j2

```diff
@@ -22,5 +22,9 @@ if {{ use_cyberglove | lower }}; then
 fi
 sed -i 's|eth_port" value=.*|eth_port" value="{{ ethercat_interface }}" />|' $(rospack find sr_ethercat_hand_config)/launch/sr_rhand.launch
 if {{ launch_hand | lower }}; then
-    roslaunch sr_ethercat_hand_config sr_rhand.launch 2> >(tee -a ~/.ros/log/stderr.log >&2)
+    roslaunch sr_ethercat_hand_config sr_rhand.launch 2> >(tee -a /home/user/.ros/log/stderr.log >&2)
+fi
+if {{ use_steamvr }}; then
+      cd /home/user/.steam/bin
```

Comment:
hm I am not sure if we can have both here: autostarting the hand and the steamvr at the same window?
Maybe if we start steamvr first in thread mode?

Reply:
I have fixed this in the latest commits